modules/Hoot/managers/api.js

```diff
@@ -1198,42 +1198,15 @@ export default class API {
             .then( resp => {
                 return {
                     data: resp.data,
-                    message: 'Conflate differential has succeeded.',
+                    message: 'Derive changeset has succeeded.',
                     status: 200,
                     type: 'success'
                 };
             } )
             .catch( err => {
                 return {
                     data: err.data,
-                    message: 'Error doing Conflate differential!',
-                    status: err.status,
-                    type: 'error'
-                };
-            } );
-    }
-
-    conflationUpload( data ) {
```

Comment:
Did you mean to delete this method?

Reply:
yup. left over code from before we cleaned up the grail functions